### PR TITLE
Make Terraform Module

### PR DESCRIPTION
**Pull request by smiller171**

### CODE REVIEW

File: tmp/app.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function
```
Comment (cdaniluk) [inline tmp/app.py]: how did this file sneak in?

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+variable "memory_size" {
+  default     = 512
```
Comment (cdaniluk) [inline variables.tf]: lotta memory for a little lambda, no?

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    ]
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "lambda" {
```
Comment (cdaniluk) [inline main.tf]: since it's unambiguous, this should be `this`

File: main.tf [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+  tags = merge(
+    var.tags,
+    {
+      "Name" = "${var.name}"
+    }
+  )
+}
+
+resource "aws_lambda_permission" "allow_S3_notification" {
```
Comment (cdaniluk) [inline main.tf]: since it's unambiguous, this should be `this`

**Review by cdaniluk — CHANGES_REQUESTED** [review]
minor things, looks good

File: variables.tf (hunk 2 above)
Comment (smiller171) [reply variables.tf]: I just grabbed the setting from upstream

File: tmp/app.py (hunk 1 above)
Comment (smiller171) [reply tmp/app.py]: gah, thanks

File: variables.tf (hunk 2 above)
Comment (smiller171) [reply variables.tf]: memory set vs memory used:
`512 / 67`

I can definitely reduce

**Comment by smiller171** [comment]
@sblack4 You're more than welcome to add your hooks in for this module

**Review by cdaniluk — APPROVED** [review]

**Pull request merged**